mycli/main.py

```diff
@@ -799,7 +823,10 @@ def format_output(self, title, cur, headers, expanded=False,
                 formatted = self.formatter.format_output(
                     rows, headers, format_name='vertical')
 
-            output.append(formatted)
+            if not isinstance(formatted, types.GeneratorType):
+                formatted = formatted.splitlines()
```

Comment:
@meeuw, so if I understand this correctly, this line won't be needed once the CLI helpers changes are made, right?